tests/etl/test_perf_data_adapters.py

```diff
@@ -63,16 +63,14 @@ def test_adapt_and_load(self):
                 if talos_datum.get('summary'):
                     # if we have a summary, ensure the subtest summary values made
                     # it in
-                    for measure in ['min', 'max', 'std', 'mean', 'median']:
-                        self.assertEqual(
-                            round(talos_datum['summary']['subtests'][testname][measure], 2),
-                            perfdata[measure])
+                    self.assertEqual(
+                        round(talos_datum['summary']['subtests'][testname]['filtered'], 2),
+                        perfdata['value'])
                 else:
                     # this is an old style talos blob without a summary. these are going
```

Comment:
Unfortunately the Android Talos tests don't have this summary information, so we still need to process this type of information from them as long as they're turned on. Hopefully won't be that much longer. After that, I'll be happy to clean this up. :)